apps/frontend/src/libs/components/button/button.tsx

```diff
@@ -1,24 +1,36 @@
 import { NavLink } from "react-router-dom";
 
+import { getValidClassNames } from "~/libs/helpers/helpers.js";
+
 import styles from "./styles.module.css";
 
 type Properties = {
+	className?: string;
+	disabled?: boolean;
 	href?: string | undefined;
 	label: string;
 	type?: "button" | "submit";
 };
 
-const Button = ({ href, label, type = "button" }: Properties): JSX.Element => {
+const Button = ({
+	className = "",
```

Comment:
Adding `className` prop to common components is not good idea. In case you want to control the size of the button, you can do this with wrapper classes. Check this [comment](https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/issues/87#issuecomment-2316856589). Also you can check my [PR](https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/115) where I also modify this component and how I use it in `ProjectCreateForm`

Reply:
got it